css/style.css

```diff
@@ -6,6 +6,118 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+.logo {
+    width: 45px;
+    height: 45px;
+    float: left;
+    padding-left: 130px;
+}
+
+ul {
+    list-style: none;
+}
+
+
+li {
+    display: inline-block;
+    padding: 10px;
+    margin-left: 25px;
+}
+
+header {
+    justify-content: space-between;
+    text-align: center;
+    font-size: 16px;
+    color: darkslategray;
+
+}
+
+.first-banner {
+    max-width: 100%;
+    padding-bottom: 15px;
+}
+
+.banner {
+    position: relative;
+    text-align: center;
+    color: white;
+}
+
+.introduction {
+    position: absolute;
```

Comment:
You shouldn't have to use absolute positioning here. 
If you switch to setting the large image as a `background-image` in CSS instead of using an `img` tag, then you'll be able to superimpose the text on top of it much more easily. 

Once you've done the image switch, change the positioning for `introduction` and `intro-subheading` so it uses flexbox instead.